public/js/user/requestDom.js

```diff
@@ -42,7 +74,18 @@ next.addEventListener('click', (e) => {
   }
   pesonalSection.classList.add('hide');
   requestSection.classList.remove('hide');
+  firstTab.classList.replace('nav--form-personalDetails', 'nav--form-yourRequest');
+  secondTab.classList.replace('nav--form-yourRequest', 'nav--form-personalDetails');
```

Comment:
@ahmedisam99 Is this classList replace logic intended to make one tab appear "selected" / "highlighted"?  If so, it might be more clear if those class names were updated (so for example, the base styles went into a class like `.nav--form-tab` and then you toggle a `selected` class on them.

Reply:
@tomduggan85 
it causes me some problems when I do it that way (i suffer with CSS :\)